frontend/src/components/SettingsAccount.vue

```diff
@@ -214,11 +241,17 @@ const actuallyDeleteAccount = async () => {
       </label>
       <label class="mt-4 flex items-center pl-4">
         <div class="w-full max-w-2xs">{{ t('label.displayName') }}</div>
-        <input
-          v-model="activeDisplayName"
-          type="text"
-          class="w-full rounded-md"
-        />
+        <div class="w-full">
+          <input
+            v-model="activeDisplayName"
+            type="text"
+            class="w-full rounded-md"
+            :class="{ '!border-red-500': errorDisplayName }"
```

Comment:
add :required="true"

Reply:
Is the display name actually required? I thought it was optional?

Reply:
Ah misread the field. Meant to add it under activeUsername

Reply:
Ah alright, that makes sense, thank you!

Reply:
The problem here is, that it triggers the modal and does request the backend, giving a less accurate error message:

![image](https://github.com/user-attachments/assets/1cd249cb-4f5d-46e7-a2c9-9d89470efeb5)